package.json

```diff
@@ -14,7 +14,11 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test"
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
```

Comment:
why adding the "eject" script again if we deleted it? We agreed that we don't need it.